go/adbc/driver/databricks/stmt_record_reader.go

```diff
@@ -69,7 +69,7 @@ func (c *chunkResponse) Next() bool {
 	return c.reader.Next()
 }
 
-type reader struct {
+type stmt_reader struct {
```

Comment:
The go convention for private type names is camelCase: stmtReader. Similar to `connectionImpl`, `databaseImpl`...